### DISCUSSION TITLE
Effects of subjecting oscilloscope to transient over-voltage (>20v)

### DISCUSSION THREAD

**Discussion by MatthewSteeves** [espotek-org/Labrador#279] [General]
Forgive my ignorance. What ill effects (if any) will happen if the oscilloscope sees voltage transients greater than 20V?

Background: 
I'm trying to measure voltage across a knife switch as it's opening. The power source for the circuit is a TIG welder. During welding it outputs about 17v max before the arc goes out (as measured with a DMM). My understanding is that during a switch opening, voltage could spike significantly higher than that. I'd like to use the Labrador to get a better picture then a DMM gives, but don't want to ruin it doing so. :D

**Comment by EspoTek** [comment]
>What ill effects (if any) will happen if the oscilloscope sees voltage transients greater than 20V?

The cutoff voltage is slightly higher than 20V (I think it might be around 24V, IIRC?), but if you exceed this then the blue smoke will be released from either the main CPU, or (if you're lucky) one of the input resistors.
The host PC should never get damaged as the USB spec required OCP.  Have never heard of it happening either.

Either way, probably best to not directly wire it up to a TIG welder.
